span_marker/model_card_template.md

```diff
@@ -0,0 +1,174 @@
+---
+# For reference on model card metadata, see the spec: https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1
+# Doc / guide: https://huggingface.co/docs/hub/model-cards
+{{ card_data }}
+---
+
+# {{ model_name | default("SpanMarker for Named Entity Recognition", true) }}
+
+This is a [SpanMarker](https://github.com/tomaarsen/SpanMarkerNER) model{% if dataset_id %} trained on the [{{ dataset_name if dataset_name else dataset_id }}](https://huggingface.co/datasets/{{ dataset_id }}) dataset{% endif %} that can be used for {{ task_name | default("Named Entity Recognition", true) }}.{% if encoder_id %} This SpanMarker model uses [{{ encoder_name if encoder_name else encoder_id }}](https://huggingface.co/models/{{ encoder_id }}) as the underlying encoder.{% endif %}
+
+## Model Details
+
+### Model Description
+
+- **Model Type:** SpanMarker
+{% if encoder_id -%}
+    - **Encoder:** [{{ encoder_name if encoder_name else encoder_id }}](https://huggingface.co/models/{{ encoder_id }})
+{%- else -%}
+    <!-- - **Encoder:** [Unknown](https://huggingface.co/models/unknown) -->
+{%- endif %}
+- **Maximum Sequence Length:** {{ model_max_length }} tokens
+- **Maximum Entity Length:** {{ entity_max_length }} words
+{% if dataset_id -%}
+    - **Training Dataset:** [{{ dataset_name if dataset_name else dataset_id }}](https://huggingface.co/datasets/{{ dataset_id }})
+{%- else -%}
+    <!-- - **Training Dataset:** [Unknown](https://huggingface.co/datasets/unknown) -->
+{%- endif %}
+{% if language -%}
+    - **Language{{"s" if language is not string and language | length > 1 else ""}}:**
+    {%- if language is string %} {{ language }}
+    {%- else %} {% for lang in language -%}
+            {{ lang }}{{ ", " if not loop.last else "" }}
+        {%- endfor %}
+    {%- endif %}
+{%- else -%}
+    <!-- - **Language:** Unknown -->
```

Comment:
This is very cool :)

Reply:
Haha, it took a while to get the Jinja to co-operate here.